R/format_value.R

```diff
@@ -120,13 +120,36 @@ check_aligns <- function(algn) {
 #' @return A formatting function which wraps and will apply the specified \code{printf} style format
 #'   string \code{format}.
 #' @seealso \code{\link[base]{sprintf}}
+#' 
+#'  \code{\link[base]{round_fmt}}
```

Comment:
we use only `[round_fmt()]`. It comes from formatters too, not base

Reply:
use the above not \link

Reply:
I've edited this, thank you for the feedback, pls lmk if it needs any further amendment.